src/data-access/MwApiLexemeCreator.ts

```diff
@@ -37,10 +45,32 @@ export default class MwApiLexemeCreator implements LexemeCreator {
 				language: lexemeLanguageItemId,
 				lexicalCategory: lexicalCategoryItemId,
 			} ),
+			errorformat: 'html',
+			formatversion: 2,
 		} );
 		const response = await this.api.postWithEditToken( params )
-			.catch( () => {
-				return Promise.reject( [ { type: 'unknown' } ] );
+			// jQuery promises can resolve and reject with multiple values;
+			// the third argument is either missing or the result
+			.catch( ( code: string, _?: unknown, result?: ApiResult ): Promise<never> => {
+				let errors: SubmitError[];
+				try {
+					if ( result && Array.isArray( result.errors ) ) {
```

Comment:
Maybe this should just be `result && result.errors`? If `result.errors` exists, but is not an array, then I feel like this should go through the `catch` block (`type: 'assertionerror'`) rather than the `else` block (`type: code`) – after all, the API has responded in a highly unexpected way in that case.

Reply:
Yeah, that makes sense to me.

Reply:
Alright, thanks. Pushed.